server/vcr-server/api/v2/models/Credential.py

```diff
@@ -67,13 +67,16 @@ def get_remote_name(self):
     def all_names(self):
         return self._cached("names", self.names.all())
 
+    # DEPRECATED
```

Comment:
Is there a better way to mark this property as deprecated, so that it will actually cause a warning when used/referenced?

Reply:
Tbh we could just remove this too. It's not actually referenced anywhere